docs/design/network-discovery/disable-port-discovery/domain-model.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+
+title Kilda Domain Model
+
+class Switch {
+   id : SwitchId
+
+   ...
+   __
+   unique constraint on switch_id
+}
+
+class PortProperties {
+   switch : Switch
+   port_no
+   discoveryEnabled
```

Comment:
Please, be consistent - either snake_case or camelCase.

Reply:
Fixed